src/test/regress/expected/pg12.out

```diff
@@ -361,8 +361,60 @@ SELECT DISTINCT y FROM test;
  25
 (1 row)
 
+-- non deterministic collations
+CREATE COLLATION case_insensitive (
+	provider = icu,
+	locale = 'und-u-ks-level2',
+	deterministic = false
+);
+\c - - - :worker_1_port
+CREATE COLLATION test_pg12.case_insensitive (
+	provider = icu,
+	locale = 'und-u-ks-level2',
+	deterministic = false
+);
+\c - - - :worker_2_port
+CREATE COLLATION test_pg12.case_insensitive (
+	provider = icu,
+	locale = 'und-u-ks-level2',
+	deterministic = false
+);
+\c - - - :master_port
+set search_path to test_pg12;
+CREATE TABLE col_test (
+	id text collate case_insensitive,
+	val text collate case_insensitive,
+	key int
+);
+insert into col_test values
+	('asdf','vaLue', 1), ('Asdf','vAlue', 2), ('asDF','valUE', 3);
+select create_distributed_table('col_test', 'id');
+NOTICE:  Copying data from local table...
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+insert into col_test values
+	('aSdf','vALue', 4), ('AsDf','vAluE', 5), ('asdF','value', 6);
+select count(*)
+from col_test
+where id = 'asdf' COLLATE case_insensitive;
```

Comment:
This should also use the collation for the column if we omit the `COLLATE case_insensitive` from the `SELECT`. It seems that it doesn't.